package.json

```
@@ -30,13 +30,13 @@
     "mongodb-connection-string-url": "^3.0.2"
   },
   "peerDependencies": {
-    "@aws-sdk/credential-providers": "^3.188.0",
-    "@mongodb-js/zstd": "^1.1.0 || ^2.0.0",
-    "gcp-metadata": "^5.2.0",
+    "@aws-sdk/credential-providers": "^3.806.0",
+    "@mongodb-js/zstd": "^2.0.0",
+    "gcp-metadata": "^7.0.1",
     "kerberos": "^2.0.1",
     "mongodb-client-encryption": "^7.0.0-alpha",
-    "snappy": "^7.3.2",
-    "socks": "^2.7.1"
+    "snappy": "^7.3.0",
```

```suggestion
    "snappy": "^7.3.2",
```

Are we sure we want to _lower_ this value?  It's already set to 7.3.2, and why would users want _not_ to have the latest patch for 7.3?

---

for certainly not